src/main/java/duke/Launcher.java

```diff
@@ -0,0 +1,9 @@
+package duke;
+
+import javafx.application.Application;
+
```

Comment:
Maybe you could add comments to include a description of what this class does....what do u think? 🤔